std-bits/database/src/main/java/org/enso/database/JDBCProxy.java

```diff
@@ -36,16 +38,19 @@ public static Object[] getDrivers() throws SQLException {
    * @return a connection
    */
   public static Connection getConnection(String url, Properties properties) throws SQLException {
-    initialize();
     return DriverManager.getConnection(url, properties);
   }
 
-  private static void initialize() throws SQLException {
-    if (!org.postgresql.Driver.isRegistered()) {
-      org.postgresql.Driver.register();
+  public static String[] getStringColumn(ResultSet resultSet, String column) throws SQLException {
```

Comment:
Any reason why this is implemented in Java instead of pure Enso? Seems like `Vector.new_builder` + regular polyglot calls would suffice.

Reply:
Ok, can we have a TODO then? Otherwise I'd be worried it can likely get lost. But maybe it's not that important so it's not a problem too.